news/2020-09-13-project-loved-september-2020.md

```diff
@@ -0,0 +1,245 @@
+
+[Sta - 99 Glooms](https://osu.ppy.sh/beatmapsets/961775#mania) by [Ticle](https://osu.ppy.sh/users/11602148)\
+185 BPM, 2:36 | \[4K\] 5.29★
+
+A familiar face in the Loved section, Ticle returns to voting with *99 Glooms*. Ticle has made a name for himself as a very expressive mapper, taking much advantage of flowy patterns and long note releases. *99 Glooms* is a prime example of Ticle's signature style, filled with tricky patterns that one might almost describe as slippery or smooth, which is quite reflective of the music at many points. The map pushes the player into a state of focus and concentration with the immense usage of mini-LNs, which have to be released very quickly and precisely, and the fine utilization of staggered releases throughout the map; the usage of long notes with varying lengths also creates a tension that drives the player to pay closer attention. Ticle is also shown to be fairly creative and detailed in the manners in which particular sounds are expressed -- the map often leverages short, light, and quick bursts, which articulate on the lighter nature of the instruments like the piano and guitar, and there are multiple instances towards the end where the long notes which correspond a particular synth are seen to gradually get longer to represent a crescendo.\
+Overall, *99 Glooms* is a great experience for a wide range of players, not only due to how tricky it can get even for the most experienced of long note enthusiasts, but also because of the musical detail that is expressed. It's about time that this map got its chance to shine in the Loved section.
+
+[![](/wiki/shared/news/2020-09-13-project-loved-september-2020/mania/4-luvoratorrrrry.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[GigaP - LUVORATORRRRRY!](https://osu.ppy.sh/beatmapsets/276833#mania) by [Fullerene-](https://osu.ppy.sh/users/2531335)\
+128 BPM, 3:19 | \[4K\] 5.85★, \[4K\] 5.97★
+
+Slow jacks and jumptrills, dumpy streams and bursts, and a special something spelled out in quads are the defining features of our next pick, *LUVORATORRRRRY!*, an exciting and bouncy song by Giga-P complimented very well with the chart laid out by Fullerene-! The different parts of the song are met with quite variable pattern types, with the sung vocals throughout the song taking the form of dumped streams and bursts, with the rap-esque vocals corresponding to slow jack patterns, and then jumptrills and quads to the synth buildups that happen twice. The set even includes two difficulties, so that the message spelled out in the quads can be deciphered by both upscroll and downscroll users (Upscroll players can see it in "On Top", and downscroll can see it in "Down Below"). The bouncy and variable nature of the song and chart make it no wonder why it gained its popularity in the first place, and that popularity is sure to carry over to it with its nomination to the Loved section.
```

Comment:
```suggestion
Slow jacks and jumptrills, dumpy streams and bursts, and a special something spelled out in quads are the defining features of our next pick, *LUVORATORRRRRY!*, an exciting and bouncy song by Giga-P complimented very well with the chart laid out by Fullerene-! 

The different parts of the song are met with quite variable pattern types, with the sung vocals throughout the song taking the form of dumped streams and bursts, with the rap-esque vocals corresponding to slow jack patterns, and then jumptrills and quads to the synth buildups that happen twice. The set even includes two difficulties, so that the message spelled out in the quads can be deciphered by both upscroll and downscroll users (Upscroll players can see it in "On Top", and downscroll can see it in "Down Below"). 

The bouncy and variable nature of the song and chart make it very telling as to why it became so popular in the first place, and that popularity is sure to carry over to it with its nomination to the Loved section.
```